Tools/environment_install/install-prereqs-ubuntu.sh

```diff
@@ -402,13 +406,6 @@ if [ -n "$PYTHON_VENV_PACKAGE" ]; then
     fi
 fi
 
-# try update packaging, setuptools and wheel before installing pip package that may need compilation
-SETUPTOOLS="setuptools"
-if [ ${RELEASE_CODENAME} == 'focal' ]; then
-    SETUPTOOLS=setuptools==70.3.0
-fi
-$PIP install $PIP_USER_ARGUMENT -U pip packaging $SETUPTOOLS wheel
```

Comment:
This changes our setup for *everything*, even if it isn't using venv.

Doesn't this mean we're no longer updating `wheel` and `packaging` outside of those things doing venv?

Reply:
Correct. I bundled them together because if you update wheel without updating setuptools, then that is another way to hose the environment. We want to ensure they all stay ABI compatible. Either they all go in as APT and are kept compatible by the OS, or they all stay at latest from pip.